service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -347,6 +347,24 @@ public void verifyUrgentDirectionsOrderExists(CaseData caseData, String migratio
         }
     }
 
+    public void verifyStandardDirectionOrderExists(CaseData caseData, String migrationId, UUID documentId) {
+        if (caseData.getStandardDirectionOrder() == null || isEmpty(caseData.getStandardDirectionOrder())) {
```

Comment:
I guess `isEmpty` has already checked the nullability.

Reply:
Pushed changes